pandas-stubs/_libs/tslibs/timestamps.pyi

```diff
@@ -71,9 +68,9 @@ class Timestamp(datetime):
         minute: int | None = ...,
         second: int | None = ...,
         microsecond: int | None = ...,
+        *,
         nanosecond: int | None = ...,
         tzinfo: _tzinfo | None = ...,
```

Comment:
`tzinfo` should move before the asterisk.

Reply:
I have checked with the pandas version and realigned all the arguments properly.